decision-records/2024-11-12--up-alert-rules.md

```diff
@@ -0,0 +1,64 @@
+# Up Alert Rules
+**Date:** 2024-11-12
+**Authors:** @dstathis
+
+## Context and Problem Statement
+
+Charms in the COS ecosystem currently have a wide range of different alert rules pertaining to the "up" metric. This
+particular metric exists for all scrape targets and always behaves the same. Therefore we should have a uniform set of
+alert rules.
+
+## Accepted Solution
+
+The folowing alert rule group should be used in each charm:
+
+```
+groups:
+- name: HostHealth
+  rules:
+  - alert: HostUnreachable
+    expr: up < 1
```

Comment:
Is this not a duplicate of the HostDown expression?

Reply:
Nope. Difference is the "for" field and the severity